src/js/index.js

```diff
@@ -0,0 +1,55 @@
+import Expression from "./Expression.js"
+
+
+const selectElements = (query) => {
+    return document.querySelectorAll(query);
+}
+
+const selectElement = (query) => {
+    return document.querySelector(query);
+}
+
+const setClickEvts = (elements, callback) => {
+    elements.forEach(element => {
+        element.addEventListener('click', callback())
+    });
+}
+
+window.onload = ()=>{
+    const expression = new Expression();
+
+    const total = selectElement('#total');
+    const setTotal = (text) =>{
+        total.innerText=text;
+    }
+
+    const getOperand = () =>{
+        return (event) => {
```

Comment:
코드 내부에서 event 를 사용하지 않으니깐 아래처럼 하는것도 좋아보입니다.
```suggestion
        return ({target}) => {
```